test/distributed/org/apache/cassandra/distributed/test/sai/IndexStreamingTest.java

```diff
@@ -92,10 +92,10 @@ public void testIndexComponentStreaming() throws IOException
             ));
 
             int num_components = isLiteral ? sstableStreamingComponentsCount() +
-                                             V1OnDiskFormat.PER_SSTABLE_COMPONENTS.size() +
+                                             V1OnDiskFormat.SKINNY_PER_SSTABLE_COMPONENTS.size() +
```

Comment:
I think it would be interesting to use a wide table to see how all the possible components are streamed. Even better, we could parameterise whether the table is skinny or wide: https://github.com/adelapena/cassandra/commit/481c144105b886d1f8891011810c4b3e6cd9bf1b
If the tests turns out to be too slow or it OOMs, we can always split it.

Reply:
I think this is a good idea for streaming particularly.